tests/plugins/dkim_signer.js

```diff
@@ -38,11 +38,17 @@ function getValueFromDKIM (dkim_header, key) {
             return arr[2];
         }
     }
-    throw `Key ${key} not found at ${dkim_header}`;
+    throw new Error(`Key ${key} not found at ${dkim_header}`);
 }
 
 const props = { selector: 'selector', domain: 'haraka.top', private_key: privateKey };
 
+const email = 'Ignored: header\r\n\r\nHi.\r\n\r\nWe lost the game. Are you hungry yet?\r\n\r\nJoe.\r\n';
+
+const ignoredHeader = 'Ignored: header\r\n\r\n';
```

Comment:
Why extract these from within their test cases? This certainly doesn't improve legibility.